src/model/document.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+from typing import List, Optional
+
+from src.model.actor import Actor
+from src.model.annotation import Annotation
+from src.model.external_document_ref import ExternalDocumentRef
+from src.model.extracted_licensing_info import ExtractedLicensingInfo
+from src.model.file import File
+from src.model.package import Package
+from src.model.relationship import Relationship
+from src.model.snippet import Snippet
+from src.model.version import Version
+
+
+class CreationInfo:
+    creators: List[Actor]  # min. 1 element
+    created: datetime
+    comment: Optional[str]
+
+    def __init__(self, creators: List[Actor], created: datetime, comment: Optional[str] = None):
+        self.creators = creators
+        self.created = created
+        self.comment = comment
+
+
+class Document:
+    data_license = "CC0-1.0"
+
+    # Mandatory properties
+
+    spdx_version: str
+    spdx_id: str
+    name: str
+    document_namespace: str
+    creation_info: CreationInfo
+
+    # Optional properties
+
+    external_document_references: List[ExternalDocumentRef]
+    license_list_version: Optional[Version]
```

Comment:
We seem to use the rdf part of the spec here by introducing a class `CreationInfo`. Part of this class is also the license list version.

Reply:
The reason I decided for the class was the `CreatorComment` field. I found it weird to have both `CreatorComment` and `DocumentComment` on top-level.
I think I just missed the `license_list_version` part. I'll move it